### ISSUE TITLE
Picker returns wrong "selectionIndex" value

### ISSUE THREAD

**Issue by elshadsm** [eclipsesource/tabris-js#1511]
### Problem description

During runtime,  we programmatically change Picker item selection by updating the "selectionIndex". Visually everything works okay. However, when we try to read the "selectionIndex" afterward it is updated, we get only 0 value. 

### Expected behavior

Picker must return the same "selectionIndex" value that was set before.

### Environment

- Tabris.js version: 2.1
- Device: iPhone 6, Nexus 6P
- OS: iOS 11.0, Android 8.0.0

### Code snippet

```javascript
const {Picker, ui} = require('tabris');

const AIRPORTS = [
  {
    id: 'SFO',
    name: 'San Francisco'
  },
  {
    id: 'TXL',
    name: 'Berlin Tegel'
  },
  {
    id: 'FRA',
    name: 'Frankfurt'
  }
];

let picker = new Picker({
  left: 20, top: 20, right: 20,
  itemCount: AIRPORTS.length,
  itemText: (index) => AIRPORTS[index].name
}).appendTo(ui.contentView);

ui.contentView.on('resize', () => {
  setTimeout(() => {
    picker.selectionIndex = 2;
    console.log('- ' + picker.selectionIndex);
  }, 2000);
});
```

When we run the source above, `- 0` is printed in the console. However, `- 2` must be printed.

**Comment by mpost** [comment]
I can confirm the issue on Android (and likely iOS). The bug stems from the js part. The _get_ operation to read the `selectionIndex` is send before the _set_ operation.

**Comment by cpetrov** [comment]
This issue has been fixed in 9f3bb8e62e5127120aa57a5a9ebb98f7b57d416b
